Development/ids.xsd

```diff
@@ -193,19 +175,12 @@
 				</xs:complexType>
 			</xs:element>
 		</xs:sequence>
+		<xs:attribute name="description" type="xs:string" use="optional"/> 
 	</xs:complexType>
 	<xs:complexType name="specificationType">
 		<xs:sequence>
 			<xs:element name="applicability" type="ids:applicabilityType"/>
-			<xs:element name="requirements" minOccurs="0">
-				<xs:complexType>
-					<xs:complexContent>
-						<xs:extension base="ids:requirementsType">
-							<xs:attribute name="description" type="xs:string" use="optional"/>
```

Comment:
The `requirementsType` was referenced only here, so no need to add the description via extension, I've added it directly to the type specification.